1. CWE Base Mod/Cold War Era Mod (CWE)/common/goods/00_goods.txt

```diff
@@ -953,7 +951,7 @@ copper = { #For electric boards
 
 rare_earth_minerals = { #For circuit boards
 	texture = "gfx/interface/icons/goods_icons/rare_earth_minerals.dds"
-	cost = 10
```

Comment:
Base prices should be kept uniform at 10 for raw resource, 20 for intermediate and 40 for completed products. This should not be an issue as actual prices will be determined by market forces anyway.